modules/ozoneBidAdapter.js

```diff
@@ -794,7 +1118,16 @@ export const spec = {
     objRet = this._addVideoDefaults(objRet, childConfig, true); // child config will override parent config
     return objRet;
   },
+  /**
+   * modify objRet, adding in default values
+   * @param objRet
+   * @param objConfig
+   * @param addIfMissing
+   * @return {*}
+   * @private
+   */
   _addVideoDefaults(objRet, objConfig, addIfMissing) {
+    // add inferred values & any default values we want.
```

Comment:
you cannot infer placement = 1 from instream, this was fought for by the ttd team

Reply:
Because of #10452  and changing (breaking) guidance from the IAB tech lab that is resulting in a bunch of supply getting banned by ttd policy changes

Reply:
See https://github.com/prebid/Prebid.js/issues/10406#issuecomment-1705603721

Reply:
Context = instream does not imply placement = 1 any longer. The former indicates a technical integration and the later indicates a user experience.

Reply:
Basically a breaking change has been forced on us, and we're no longer comfortable putting publisher inventory at risk of getting banned because JavaScript in this project is making an inference no longer valid. 

Video.context can no longer imply placement of plcmt, and we've removed that from our core library 

#11719